controller/work_item_link_type.go

```diff
@@ -89,8 +91,7 @@ func enrichLinkTypeList(ctx *workItemLinkContext, list *app.WorkItemLinkTypeList
 	for categoryID := range categoryIDMap {
 		modelCategory, err := ctx.Application.WorkItemLinkCategories().Load(ctx.Context, categoryID)
 		if err != nil {
-			jerrors, httpStatusCode := jsonapi.ErrorToJSONAPIErrors(err)
-			return ctx.ResponseData.Service.Send(ctx.Context, httpStatusCode, jerrors)
+			err
```

Comment:
if you want to ignore the error why don't you use `modelCategory, _ := ctx.Application.WorkItemLinkCategories()`,

BTW: Why do u do this :) ?